src/components/AuthScreen/AuthScreen.component.js

```diff
@@ -6,7 +6,7 @@ import './AuthScreen.css';
 
 class AuthScreen extends Component {
   static propTypes = {
-    intl: intlShape.isRequired
+    intl: intlShape.isRequired,
```

Comment:
Please avoid to commit this change as is this change is not neccesary for this PR ;)

Reply:
I believe this happened because Prettier got updated somehow in the branch I was working on. I'll roll this back.